app/build.gradle

```diff
@@ -0,0 +1,39 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 29
+    buildToolsVersion "30.0.0"
+
+    defaultConfig {
+        applicationId "com.fmpdroid.moneybox"
+        minSdkVersion 23
+        targetSdkVersion 29
+        versionCode 1
+        versionName "1.0"
```

Comment:
Normally, every time we create new features/functionality, we need to increase the version of our project.
You may refer to this as your guide:
https://developer.android.com/studio/publish/versioning
https://docs.gradle.org/current/userguide/single_versions.html